Revert/Sources/AlertViewController.swift

```diff
@@ -49,10 +49,8 @@ extension AlertViewController {
     case .ActionSheet:
```

Comment:
@tonyarnold Should we be removing UIAlertView and UIAlertSheet all together?  These are deprecated in iOS 8. But you could still use them in an iOS 8 app (legacy code) and Reveal will/should support those controls.